src/components/tabController/TabBar.tsx

```diff
@@ -201,8 +201,16 @@ const TabBar = (props: Props) => {
     }
   });
 
+  const tabBarItemStyle = useMemo(() =>
```

Comment:
Is it working? 
You are memoizing a function that returns a value and not the value

Reply:
I had an issue, changed it to `useCallback` and passed the `tabBarItemStyle` as dependency to the `tabBarItems`.
Checked on mobile and web.